schemas/base.json

```diff
@@ -102,6 +102,47 @@
 				}
 			}
 		},
+		"files": {
+			"$id": "#/definitions/files",
+			"type": "object",
+			"title": "Files",
+			"required": [
+				"title"
+			],
+			"properties": {
+				"title": {
+					"$id": "#/definitions/files/properties/title",
+					"type": "string",
+					"title": "File Title",
+					"description": "Title of the file",
+					"examples": [
+						"Board Reports"
+					]
+				},
+				"manifest": {
+					"$id": "#/definitions/files/properties/manifest",
+					"type": ["string"],
+					"title": "Manifest URI",
+					"description": "URI for a IIIF Manifest",
+					"examples": [
+						"https://wwww.manifest_uri.com/1234"
+					]
+				},
+				"download": {
+					"$id": "#/definitions/files/properties/download",
+					"type": ["string"],
```

Comment:
It looks like there is as `uri` type - is that a more appropriate type to use here? https://json-schema.org/understanding-json-schema/reference/string#resource-identifiers

Reply:
Sorry, I must have missed that! I’ll update it with the additional fixture for multiple files in the array.

Reply:
Obvs we'd want to use that type for the manifest as well. And then let's make sure the type checking is working by tweaking the invalid fixtures.

Reply:
OK after further conversation it looks like this is format checking (rather than type checking) which is currently not enforced. We'll leave this as a string for now and if we need to introduce custom validation later we can do that.